.travis.yml

```diff
@@ -3,12 +3,12 @@ node_js:
 - '5.8'
 install:
 - bundle install
-- npm install
+- TMPDIR=/tmp yarn install
```

Comment:
Need TMPDIR=/tmp because phantomjs-prebuilt version lower than 2.1.8  crash when the TMPDIR environment variable is not set and cause yarn to fail. (https://github.com/Medium/phantomjs/issues/738 https://github.com/karma-runner/karma-phantomjs-launcher/issues/200).